components/esp_system/Kconfig

```diff
@@ -138,6 +138,13 @@ menu "ESP System Settings"
                 for the data part (above the splitting address). The memory protection is effective
                 on all access through the IRAM0 and DRAM0 buses.
 
+        config ESP_SYSTEM_PMP_LP_CORE_RESERVE_MEM_EXECUTABLE
+            bool "Make LP core reserved memory executable in HP core"
+            depends on IDF_TARGET_ARCH_RISCV && SOC_LP_CORE_SUPPORTED && ESP_SYSTEM_PMP_IDRAM_SPLIT
+            default "n"
+            help
+                If enabled, user can run code only available in LP Core image.
```

Comment:
I'd suggest removing `only` in this sentence since the current wording reads as only the LP memory would be usable which is not necessarily how the code is above.